templates/ts-apollo-mongodb-datasync-backend/README.md

```diff
@@ -0,0 +1,47 @@
+## Graphback Apollo+MongoDB Data Synchronization Runtime Template
+
+Template showcases Graphback Data Synchronization capabilities using
+resolver layer created in-memory as opposed to codegeneration.
+
+Serverless example can be build as docker image with configurable volume for models. 
+This models will be processed at application at startup giving fully featured GraphQL Server 
+following Graphback CRUD specification.
+
+For more information please refer to: 
+
+### Running example
+
+The project has been created using `graphback`. Run the project using the following steps.
+
+- Start the database
+
+```
+docker-compose up -d
+```
+
+- Define your schema in the `model/datamodel.graphql` file. Or use the default:
+
+```
+type User {
+  id: ID!
+  name: String
+}
+```
+
+> **NOTE**: for Data Synchonization to work, you must use the `@datasync` annotation on required model(s)
```

Comment:
Let's add the annotation directly to the default model

Reply:
Cool :+1: